apps/daffio/package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "cd ../.. && ng lint daffio",
     "lint:fix": "npm run lint -- --fix",
     "dev:ssr": "ng run daffio:serve-ssr",
-    "serve:ssr": "node ../../dist/apps/daffio/server/main.js"
+    "serve:ssr": "DAFFIO_DOCS_PATH=http://localhost:4000/assets/daffio/docs/ node ../../dist/apps/daffio/server/main.js"
```

Comment:
@damienwebdev should this be committed?

Reply:
I think that's ok given that these are the defaults.